src/Components/Buttons/Button.test.js

```diff
@@ -0,0 +1,17 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import EditButton from "./EditButton";
+
+test("when edit functionality clicks", async () => {
```

Comment:
when writing test description please follow 

Should ..., Given ...

so for example in this case

Should return todo item current text, Given the user click on the edit button